permissionslocker/__init__.py

```diff
@@ -35,5 +35,5 @@
 
 async def setup(bot: Red) -> None:
     cog = PermissionsLocker(bot)
-    await cog.initialize()
-    bot.add_cog(cog)
+    await cog.cog_load()
```

Comment:
Remove this line and rename `initialize` in permissionslocker.py line.64 to `cog_load`